docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+
+* **SetFirewallConfig (#SubnetManagement)**: change the Firewall configuration in the registry (configures which boundary nodes subnet blockchain replicas will communicate with).
+
+* **UpdateNodeOperatorConfig (#NodeAdmin)**: change a node operator’s allowance in the registry.
+
+* **StopOrStartNNSCanister (#NetworkCanisterManagement)**: stop or start an NNS canister.
+
+## ICP tokens
+ICP are native utility tokens that play three key roles in the network:
+
+* **Facilitating network governance**: ICP tokens can be locked to create neurons that participate in network governance by voting, through which they can earn economic rewards.
+
+* **Production of cycles for compute**: ICP provides a source store of value that can be converted into “cycles,” which power computation in the role of fuel that is burned when it is used. The NNS converts ICP to cycles at a variable rate, so chosen to ensure users of the network can always create new cycles at approximately constant cost concerning IMF SDR, such that the cost of acquiring fuel is predictable.
+
+* **Rewarding participants**: the network provides rewards to incentivize those playing important roles that enable the network to function, including a) the provision of “voting rewards” to those participating in governance, and b) the provision of “node provider rewards” to those operating the node machines that are hosting the network.
+
+## Ledger
+The ICP ledger is hosted within the NNS and records all balances of ICP in the manner of a spreadsheet. Each row is called an “account,” which has two fields (i.e., there are two “columns”):
+
+* **Account identifier (bytes)**: a unique value that is derived from the identity of the “principal” that “controls” the account. Currently, the principal must either be (i) the owner of a public key pair, or (ii) a canister smart contract that is part of the NNS. Account identifiers are derived by hashing the concatenation of a domain separator, the principal ID, and the subaccount (or zeros if no subaccount is given).
+
+* **Balance** (positive integer, representing one-hundredth of a millionth of an ICP): the quantity of ICP assigned to the principal of the account.
+
+When the principal is a public key or Canister, they can apply the following operation to an account:
+
+* **Send**: send a portion of the ICP balance to another account. If all the ICP is sent to another account, then the sending account ceases to exist (i.e., is deleted from the ledger).
+* **Notify**: when the destination of the funds sent is the account of an NNS canister (e.g., an account of the governance canister), the sender can ask the ledger to notify the recipient canister of the incoming transfer. The recipient canister can then act on this notification. Two examples where this ability is used are creating a neuron and refreshing the stake of a neuron. These are detailed below.
```

Comment:
Is this still usable? Let's make sure with @MarioDfinity . This is not anymore how neurons are created

Reply:
No. `notify` is deprecated and may be removed in future. It should not be mentioned in the documentation.